app/Http/Controllers/Admin/RoleController.php

```diff
@@ -0,0 +1,337 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use Illuminate\Http\Request;
+use Yajra\Datatables\Datatables;
+use App\Http\Controllers\Controller;
+use App\Repositories\RoleRepository;
+use App\Repositories\PermissionRepository;
+use Validator;
+use Entrust;
+use DB;
+
+class RoleController extends Controller
+{
+    public $roleRepository;
+    public $permissionRepository;
+
+    /**
+     * Create a new controller instance.
+     *
+     * @return void
+     */
+    public function __construct(RoleRepository $roleRepository, PermissionRepository $permissionRepository)
+    {
+        $this->roleRepository = $roleRepository;
+        $this->permissionRepository = $permissionRepository;
+    }
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        return view('admin.roles.index');
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        //
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        $data = $request->all();
+
+        DB::beginTransaction();
+
+        try {
+            $rules = [
+                'name'          =>  'required|string|unique:roles',
+                'display_name'  =>  'required|string|unique:roles',
+            ];
+
+            $messages = [
+                'name.required'         =>  __('Role') . __('Required'),
+                'name.string'           =>  __('Role') . __('String'),
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.SpaceAfterComma</br>**Severity**: error</br>**File**: app/Http/Controllers/Admin/RoleController.php</br>Expected 1 space between "'name.required'" and double arrow; 9 found